docs/concepts/configuration/manage-compute-resources-container.md

```diff
@@ -135,8 +135,8 @@ When using Docker:
   [`--cpu-shares`](https://docs.docker.com/engine/reference/run/#/cpu-share-constraint)
   flag in the `docker run` command.
 
-- The `spec.containers[].resources.limits.cpu` is converted to its millicore value,
-  multiplied by 100000, and then divided by 1000. This number is used as the value
+- The `spec.containers[].resources.limits.cpu` is converted to its millicore value, and
```

Comment:
Both the original description and the updated ones are incorrect technically. Here is a snippet that is more accurate:
```
The default CPU quota period is `100ms`. 
A container cannot use more than it's fair share of CPU every `100ms`. 
The total amount of CPU time that a container can use every `100ms` is `spec.containers[].resources.limits.cpu` in millicores multiplied by `100` (minimum resolution of CPU quota is `1ms`).
```